File: Client/Models/Session.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Desktop.Shops
{
    public class Session
    {
        public bool IsLogin { get; set; } = false;
        public User CurrentUser { get; set; } = new User();
    }
}

File: Shops/Models/Session.vb
Public Class Session

    Private Shared PVT_CurrentUser As User = New User()

    Public Shared Property CurrentUser() As User
        Get
            Return PVT_CurrentUser
        End Get
        Set(ByVal value As User)
            PVT_CurrentUser = value
        End Set
    End Property

End Class
